lib/rust/ensogl/pack/js/src/runner/index.ts

```diff
@@ -238,12 +244,40 @@ export class App {
     }
 
     /** Log the message on the remote server. */
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    remoteLog(message: string, data: any) {
-        // FIXME [PB]: https://github.com/enso-org/cloud-v2/issues/359
-        // Implement remote logging. This should be done after cloud integration.
-        // Function interface is left intentionally for readability.
-        // Remove typescript error suppression after resolving fixme.
+    async remoteLog(message: string, data: any) {
+        try {
+            if (
+                this.config.options.dataCollection.value &&
+                this.remoteLoggerUrl &&
+                this.accessToken
+            ) {
+                const headers: HeadersInit = new Headers()
+                headers.set('Content-Type', 'application/json')
+                headers.set('Authorization', `Bearer ${this.accessToken}`)
+                const response = await fetch(this.remoteLoggerUrl, {
+                    method: 'POST',
+                    headers: headers,
+                    body: JSON.stringify({
+                        message: message,
+                        metadata: data,
+                    }),
+                })
+                if (!response.ok) {
+                    const errorMessage = `Error while logging message: Status ${response.status}.`
+                    try {
+                        const text = await response.text()
+                        throw new Error(`${errorMessage} Response: ${text}.`)
+                    } catch (error) {
+                        throw new Error(
+                            `${errorMessage} Failed to read response: ${String(error)}.`
+                        )
+                    }
+                }
+            }
+        } catch (error) {
+            logger.error(error)
+            throw error
+        }
     }
```

Comment:
Hmmmm, EnsoGL Pack has a logger, and you can connect any backend to it, including remote logging. For example @mwu-tow connected a file logger to it. I dont think remote logging should be part of EnsoGL Pack.